pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -311,18 +317,20 @@ func TestNewYamlConfig(t *testing.T) {
 		{
 			name: "TestNewYamlConfig",
 			args: args{
-				host:           host,
-				logoutRedirect: "",
-				brand:          DEFAULT_BRAND,
-				docURL:         DEFAULT_DOC_URL,
-				apiServerURL:   mockAPIServer,
+				host:              host,
```

Comment:
Need to test with both.
- `name: "Test generate default yaml config"`
- `name: "Test generate yaml config with custom product values"`